core/src/main/java/bio/terra/pearl/core/service/workflow/TriggerActionService.java

```diff
@@ -68,8 +69,17 @@ public void handleEvent(EnrolleeEvent event) {
 
         for (Trigger trigger: applicableTriggers) {
             if (TriggerActionType.NOTIFICATION.equals(trigger.getActionType())) {
-                notificationDispatcher.dispatchNotificationAsync(trigger, event.getEnrolleeContext(),
-                        event.getPortalParticipantUser().getPortalEnvironmentId());
+                //if event is EnrolleeSurveyEvent, we need to check if the event was a survey
```

Comment:
this _could_ have been a new event type like `SURVEY_RESPONSE_UPDATED`, but given that autosave triggers a survey response update every few seconds, i really don't think there's a good way for an admin to capitalize on an event type like that